app/javascript/admin/__tests__/controllers/config_controller.test.js

```diff
@@ -1,34 +0,0 @@
-import { Application } from '@hotwired/stimulus';
-import ConfigController from '../../controllers/config_controller';
-
-describe('ConfigController', () => {
```

Comment:
Removed this test file as we are using preact-modal now and similar to other instances I have added cypress tests for checking this entire functionality.